packages/components/src/components/Divider/__tests__/divider.test.tsx

```diff
@@ -0,0 +1,77 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { render, screen } from '@testing-library/react';
+import { describe, expect, test } from 'vitest';
+
+import { Divider, DividerHeight, DividerStyle } from '../Divider';
+
+const DEFAULT_COLOR_HEX = '#CCC';
+const COLOR_HEX = '#4065AE';
+const DIVIDER_SELECTOR = 'fondue-divider';
+const DIVIDER_LINE_SELECTOR = 'fondue-divider-line';
+
+describe('Divider component', () => {
+    test('loads and displays divider', async () => {
+        render(<Divider />);
+        const divider = await screen.findByTestId(DIVIDER_LINE_SELECTOR);
+        expect(divider).toHaveStyle({ borderTopColor: DEFAULT_COLOR_HEX });
+    });
+    test('has correct color', async () => {
+        render(<Divider color={COLOR_HEX} />);
+        const divider = await screen.findByTestId(DIVIDER_LINE_SELECTOR);
+        expect(divider).toHaveStyle({ borderTopColor: COLOR_HEX });
+    });
+    test('allows the height to be set to Small', async () => {
+        render(<Divider height={DividerHeight.Small} />);
+        const divider = await screen.findByTestId(DIVIDER_SELECTOR);
+        expect(divider).toHaveStyle({ height: DividerHeight.Small });
+    });
+
+    test('allows the height to be set to Medium', async () => {
+        render(<Divider height={DividerHeight.Medium} />);
+        const divider = await screen.findByTestId(DIVIDER_SELECTOR);
+        expect(divider).toHaveStyle({ height: DividerHeight.Medium });
+    });
+
+    test('allows the height to be set to Large', async () => {
+        render(<Divider height={DividerHeight.Large} />);
+        const divider = await screen.findByTestId(DIVIDER_SELECTOR);
+        expect(divider).toHaveStyle({ height: DividerHeight.Large });
+    });
+
+    test('allows the divider border style to be dashed', async () => {
+        render(<Divider style={DividerStyle.Dashed} />);
+        const divider = await screen.findByTestId(DIVIDER_LINE_SELECTOR);
+        expect(divider).toHaveClass('tw-border-dashed');
```

Comment:
```suggestion
        expect(divider).toHaveStyle({ borderStyle: 'dashed' });
```
?

Reply:
tailwind is not evaluated in the tests, thus, only the classes are present